src/stylesheets/elements/_links.scss

```diff
@@ -117,7 +117,7 @@ body > footer a  {
     $grey-1 		        // a:hover color
   );
   &:link {
-    padding: $base-spacing-unit-tiny 3px;
+    padding: $base-spacing-unit-tiny $base-spacing-unit-medium;
```

Comment:
To make the styling of this look the same as what's currently in beta, I was thinking it might be ideal to do something like this:

```
.navigation--main a {
  ...
  &:link {
    padding: $base-spacing-unit-tiny $base-spacing-unit;
    margin-right: $base-spacing-unit-tiny;
    margin-left: -($base-spacing-unit);

    @include media($desktop) {
      margin-right: $base-spacing-unit-large;
    }
  }
  ...
}
```